pkg/apis/apps/v1beta1/types_swagger_doc_generated.go

```diff
@@ -220,12 +231,27 @@ func (StatefulSetSpec) SwaggerDoc() map[string]string {
 
 var map_StatefulSetStatus = map[string]string{
 	"":                   "StatefulSetStatus represents the current state of a StatefulSet.",
-	"observedGeneration": "observedGeneration is the most recent generation observed by this StatefulSet.",
-	"replicas":           "replicas is the number of actual replicas.",
+	"observedGeneration": "observedGeneration is the most recent generation observed for this StatefulSet. It corresponds to the StatefulSet's generation, which is updated on mutation by the API Server.",
+	"replicas":           "replicas is the number of Pods created by the StatefulSet controller.",
+	"readyReplicas":      "readyReplicas is the number of Pods created by the StatefulSet controller that have a Ready Condition.",
+	"currentReplicas":    "currentReplicas is the number of Pods created by the StatefulSet controller from the StatefulSet version indicated by currentRevision.",
+	"updatedReplicas":    "updatedReplicas is the number of Pods created by the StatefulSet controller from the StatefulSet version indicated by updateRevision.",
+	"currentRevision":    "currentRevision, if not empty, indicates the version of the StatefulSet used to generate Pods in the sequence [0,currentReplicas).",
+	"updateRevision":     "updateRevision, if not empty, indicates the version of the StatefulSet used to generate Pods in the sequence [replicas-updatedReplicas,replicas)",
 }
 
 func (StatefulSetStatus) SwaggerDoc() map[string]string {
 	return map_StatefulSetStatus
 }
 
+var map_StatefulSetUpdateStrategy = map[string]string{
+	"":          "StatefulSetUpdateStrategy indicates the strategy that the StatefulSet controller will use to perform updates. It includes any additional parameters necessary to perform the update for the indicated strategy.",
+	"type":      "Type indicates the type of the StatefulSetUpdateStrategy.",
+	"partition": "Partition is used to communicate the ordinal at which to partition the StatefulSet when Type is PartitionStatefulSetStrategyType. This value must be set when Type is PartitionStatefulSetStrategyType, and it must be nil otherwise.",
```

Comment:
@krmayankk this file is generated, but I get your meaning. However, you should add review to the comments in types.go so they are actionable.